doc/source/_toc.yml

```
@@ -74,15 +74,40 @@ parts:
         title: Ray Train
         sections:
           - file: train/getting-started
+            title: "Getting Started"
           - file: train/key-concepts
+            title: "Key Concepts"
           - file: train/user-guides
+            title: "User Guides"
             sections:
               - file: train/config_guide
+                title: "Configuring Ray Train"
               - file: train/dl_guide
+                title: "Deep Learning Guide"
               - file: train/gbdt
+                title: "Boosted Tree Guide"
```

Boosted Tree may not be well-known parlance.

```suggestion
                title: "XGBoost/LightGBM guide"
```